support/cloud-client/src/main/java/com/gcpsupport/samples/CreateCase.java

```diff
@@ -0,0 +1,134 @@
+
+    }
+
+    // helper method will return a CloudSupport object which is required for the
+    // main API service to be used.
+    private static CloudSupport getCloudSupportService() {
+
+        try {
+
+            JsonFactory jsonFactory = GsonFactory.getDefaultInstance();
+            HttpTransport httpTransport = GoogleNetHttpTransport.newTrustedTransport();
+
+            InputStream credentialsInputStream = new FileInputStream(new File(PRIVATE_KEY_JSON_PATH));
+            
+            GoogleCredentials credentials = ServiceAccountCredentials
+                                            .fromStream(Objects.requireNonNull(credentialsInputStream))
+                                            .createScoped(Collections.singletonList(CLOUD_SUPPORT_SCOPE));
+
+            HttpRequestInitializer requestInitializer = new HttpCredentialsAdapter(credentials);
+
+            return new CloudSupport.Builder(httpTransport, jsonFactory, requestInitializer).build();
+
+        } catch (IOException e) {
+            System.out.println("IOException caught in getCloudSupportService()! \n" + e);
+
+        } catch (GeneralSecurityException e) {
+            System.out.println("GeneralSecurityException caught in getCloudSupportService! \n" + e);
+        }
+
+        return null;
+
+    }
+
+    // helper method to get a CloudSupportCase object
+    private static CloudSupportCase getCloudSupportCaseJsonObject(String jsonPathName) throws IOException {
+        JsonFactory jsonFactory = GsonFactory.getDefaultInstance();
+        JsonObjectParser parser = new JsonObjectParser(jsonFactory);
+        InputStream stream = new FileInputStream(new File(jsonPathName));
+        Reader reader = new InputStreamReader(stream, "UTF-8");
+        return parser.parseAndClose(reader, CloudSupportCase.class);
+    }
+
+    // this helper method is used for createCase()
+    private static void listValidClassifcations() {
+
+        try {
+            CloudSupport supportService = getCloudSupportService();
+
+            SearchCaseClassificationsResponse request = supportService.caseClassifications().search().execute();
+
+            for (Map.Entry<String, Object> entry : request.entrySet()) {
+                System.out.println("Key = " + entry.getKey() +
+                        ", Value = " + entry.getValue());
+            }
+
+        } catch (IOException e) {
+            System.out.println("IOException caught in listValidClassifications()! \n" + e);
+        }
+
+    }
+
+    // create a new case
+    public static void createCase(String parentResource, String newCasePath) {
+
+        try {
+
+            CloudSupport supportService = getCloudSupportService();
+
+            CloudSupportCase newContent = getCloudSupportCaseJsonObject(newCasePath);
+
+            CloudSupportCase newCaseResponse = supportService.cases().create(parentResource, newContent).execute();
+
+            System.out.println("Created case's name is: " + newCaseResponse.getName());
```

Comment:
Not sure these need newlines, but I could see that being easier to read.  Your line length might be too much.